jenkins_jobs/run_host_os_admin_commands/script.sh

```diff
@@ -0,0 +1,26 @@
+VERSIONS_REPO_DIR=$(basename $VERSIONS_REPO_URL .git)
+
+
+# Fetch pull requests in case this job was triggered by one
+git clone $VERSIONS_REPO_URL $VERSIONS_REPO_DIR --no-checkout
+pushd $VERSIONS_REPO_DIR
+git fetch origin +refs/pull/*:refs/remotes/origin/pr/*
+popd
+
+eval python host_os.py \
+     --verbose \
+     release-notes \
+         --build-versions-repository-url $VERSIONS_REPO_URL \
+         --build-version $VERSIONS_REPO_COMMIT \
```

Comment:
We're not gonna want those commits to be pushed. Use `--no-push-updates` for all commands.